relnotes/meta-istring.feature.md

```diff
@@ -1,6 +0,0 @@
-* `ocean.meta.types.Qualifiers`
-
-  Moved `istring`/`cstring`/`mstring` definitions from `ocean.transition`
-  module, keeping them available in original module via non-deprecated alias.
-  This is done as part of splitting `ocean.transition` into smaller parts and
-  thus simplifying dependency chains.
```

Comment:
It's a bit weird to add files after files only to remove them in a later commit... Can't the text just be appended to meta.feature.md ?